bio2zarr/vcf.py

```diff
@@ -940,70 +981,108 @@ def convert_partition(
                     # is that we get a significant pause at the end of the counter while
                     # all the "small" fields get flushed. Possibly not much to be done about it.
                     core.update_progress(1)
-
+                partition_metadata = {
+                    "num_records": num_records,
+                    "field_summaries": {k:v.asdict() for k,v in tcw.field_summaries.items()}
+                }
+                with open(out_path / f"partition_{partition_index}_metadata.json", "w") as f:
```

Comment:
We've used ``p{j}`` as the partition file name elsewhere, may as well stick with this, ``e.g. p0_summary.json``